TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/CsvLogging.kt

```diff
@@ -64,7 +64,7 @@ object CsvLogging : API() {
             val writer = this.fileHash[file]!!
 
             writer.write("${opMode.runtime}")
-            for (i in data) fileHash[file]!!.write(", $i")
+            for (i in data) CsvLogging.fileHash[file]!!.write(", $i")
```

Comment:
```suggestion
            for (i in data) fileHash[file]!!.write(", $i")
```

Unless I'm missing something, you shouldn't need to qualify this path.